torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2111,10 +2111,31 @@ Tensor symeig_backward(const std::vector<torch::autograd::Variable> &grads, cons
 
   auto vh = v.conj().transpose(-2, -1);
 
+  // Based on a little experiment, min_threshold should be (approximately)
+  //   "max_abs_eigenvalue * (10 + matrix_size) * machine_eps".
```

Comment:
Could you please confirm this for large matrices?

Reply:
Yes, I redid the experiment [here](https://colab.research.google.com/drive/1wyt3357g8J91PQggLh8CLgvm4RiBmjQv?usp=sharing) (the previous one was flawed).